server/schema.py

```diff
@@ -276,6 +276,15 @@ class Config:
         model_fields = "__all__"
 
 
+class TeamMinSchema(ModelSchema):
+    name: str
+    slug: str
```

Comment:
dont need this when defining it in config anyway?

Reply:
yes, forgot to remove lol